packages/common/src/TruffleConfig.js

```diff
@@ -36,7 +36,15 @@ const gasPx = 20000000000; // 20 gwei
 const gas = undefined; // Defining this as undefined (rather than leaving undefined) forces truffle estimate gas usage.
 
 // If a custom node URL is provided, use that. Otherwise use an infura websocket connection.
-const nodeUrl = customNodeUrl || `wss://${name}.infura.io/ws/v3/${infuraApiKey}`;
+function getNodeUrl(networkName = argv.network) {
+  if (isPublicNetwork(networkName) && !networkName.includes("fork")) {
+    const infuraApiKey = process.env.INFURA_API_KEY || "e34138b2db5b496ab5cc52319d2f0299";
```

Comment:
side comment: do we want to cycle to a new public key for infura? is this a common one that other projects use or where did this key come from?